src/groovy/org/pih/warehouse/requisition/RequisitionStatus.groovy

```diff
@@ -157,6 +158,15 @@ enum RequisitionStatus {
             "end)"
     }
 
+    static Closure mapStatusCodes = { RequisitionStatus status ->
```

Comment:
I am wondering if we should specify what we are mapping to exactly.
Since we are mapping it to options maybe this should be called something like `mapToOption` or `toOption` and we can omit the "StatusCode" in the name since we are going to be calling it on RequisitionStatus so saying that we are mapping it to StatusCodes seems redundant, but that just my take.

To me this felts similar to what we are doing when mapping to JSON `.toJson()`

Reply:
Good idea, thanks